recipes/VoxPopuli/Tokenizer/train.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env/python3
+"""Recipe for training a BPE tokenizer with VoxPopuli
+The tokenizer converts words into sub-word units that can
+be used to train a language (LM) or an acoustic model (AM).
+When doing a speech recognition experiment you have to make
+sure that the acoustic and language models are trained with
+the same tokenizer. Otherwise, a token mismatch is introduced
+and beamsearch will produce bad results when combining AM and LM.
+
+Authors
+ * Titouan Parcollet 2024
+"""
+
+import sys
+import speechbrain as sb
+from hyperpyyaml import load_hyperpyyaml
+from speechbrain.utils.distributed import run_on_main
+
+if __name__ == "__main__":
+
+    # CLI:
+    hparams_file, run_opts, overrides = sb.parse_arguments(sys.argv[1:])
+    with open(hparams_file) as fin:
+        hparams = load_hyperpyyaml(fin, overrides)
+
+    # create ddp_group with the right communication protocol
+    sb.utils.distributed.ddp_init_group(run_opts)
+
+    # 1.  # Dataset prep (parsing Librispeech)
+    from voxpopuli_prepare import prepare_voxpopuli  # noqa
+
+    # Create experiment directory
+    sb.create_experiment_directory(
+        experiment_directory=hparams["output_folder"],
+        hyperparams_to_save=hparams_file,
+        overrides=overrides,
+    )
+
+    # multi-gpu (ddp) save data preparation
+    run_on_main(
+        prepare_voxpopuli,
+        kwargs={
+            "data_folder": hparams["data_folder"],
+            "save_folder": hparams["output_folder"],
+            "skip_prep": hparams["skip_prep"],
```

Comment:
same as before. you'll need to put more input args imho

Reply:
Nope, it will work for any language.